src/ape/api/address.py

```diff
@@ -22,15 +35,32 @@ def provider(self) -> ProviderAPI:
 
     @provider.setter
     def provider(self, value: ProviderAPI):
+        """
+        Sets the active provider if connected to one.
+
+        Args:
+            value (:class:`~ape.api.providers.ProviderAPI`): The provider to set.
+        """
+
         self._provider = value
 
     @property
     @abstractmethod
     def address(self) -> AddressType:
-        ...
+        """
+        The raw address type.
```

Comment:
The address of this account
```suggestion
        The address of this account. Subclasses must override and provide this value.
```

Reply:
Should we use the word `account` here though?
The `AddressAPI` can also represent a contract, is that considered an account?

Reply:
Ah, well based on the comment above, I see that contracts must be commonly also referred to as an accounts